frontend/src/control-flow.js

```diff
@@ -93,9 +93,15 @@ export default function InitializeControlFlow() {
     y
   );
   const lattice = new QubitLattice(workspace, app);
+  const cancelQubitConstellationButton = new Button(
+    'Cancel Qubit Constellation',
+    x,
+    y
+  );
   createQubitConstellationButton.on('click', () => {
     workspace.removeChild(createQubitConstellationButton);
     workspace.addChild(saveQubitConstellationButton);
+    workspace.addChild(cancelQubitConstellationButton);
```

Comment:
This really should be implemented as part of the undo and redo buttons we have planned.

Reply:
Related: #152